test/support/test_config.ex

```diff
@@ -0,0 +1,14 @@
+defmodule Ravenx.Test.TestConfig do
+
+  def test_multiple (payload) do
```

Comment:
It doesn't affect anything, but payload is never used and there is an space between function name and parenthesis (in the next function too).